http.go

```diff
@@ -86,6 +102,58 @@ func (s *LumbermillServer) serveHealth(w http.ResponseWriter, r *http.Request) {
 	w.WriteHeader(http.StatusOK)
 }
 
+func (s *LumbermillServer) serveInfluxDBHealth(w http.ResponseWriter, r *http.Request) {
+	errorCount := 0
+
+	// Copy the map so we can unlock.
+	s.recentTokensLock.RLock()
+	tokenMap := make(map[string]string)
+	for name, token := range s.recentTokens {
+		tokenMap[name] = token
+	}
+	s.recentTokensLock.RUnlock()
+
+	for name, token := range tokenMap {
+		// Ugh. For now. Should reuse connections, but tied up in Posters, and abstraction.
+		clientConfig := createInfluxDBClient(name, os.Getenv("INFLUXDB_SKIP_VERIFY") == "true")
+		client, _ := influx.NewClient(&clientConfig)
+
+		// Query the last point for the token and ensure it's been published to in the last XXX minutes.
+		for _, prefix := range influxDbSeriesCheckPrefixes {
+			series := prefix + token
```

Comment:
Maybe pull out some of this into another function that returns the error and increment the counter / error in the loop based on the results instead of the nested / duplicate errorCount++ ?